scripts/pylint_custom_plugin/tests/test_pylint_custom_plugins.py

```diff
@@ -1886,9 +1930,15 @@ def test_ignores_methods_return_ItemPaged(self):
         
         class SomeClient(): #@
             def list_thing(self): #@
+                '''
+                :rtype: ItemPaged()
```

Comment:
I think more commonly we'd see `~azure.core.paging.ItemPaged` and `~azure.core.async_paging.AsyncItemPaged`. We also probably want to check for when the result type is included, like `~azure.core.paging.ItemPaged[ModelOperationInfo]`